src/share/poudriere/common.sh

```diff
@@ -2090,7 +2090,7 @@ _real_build_port() {
 		# Don't need to install if only making packages and not
 		# testing.
 		[ -n "${PORTTESTING}" ] && \
-		    install_order="${install_order} install-mtree install"
+		    install_order="${install_order} test install-mtree install"
```

Comment:
Well, maybe tests need to run after the port has been really installed, and not just staged, I don't know, I'm just saying this is coherent with what bsd.port.mk does :-)